stellar/operators.js

```diff
@@ -10,20 +10,57 @@ async function processCommand(options, _, chain) {
 
     const contract = new Contract(options.address || chain.contracts?.axelar_operators?.address);
 
-    let operation;
+    let operator, operation;
 
-    const address = Address.fromString(options.args || wallet.publicKey());
+    if (['is_operator', 'add_operator', 'remove_operator'].includes(options.action)) {
+        if (!options.args) throw new Error(`Missing --args operatorAddress the params.`);
+        operator = Address.fromString(options.args).toScVal();
+    } else {
+        operator = Address.fromString(wallet.publicKey()).toScVal();
+    }
 
     switch (options.action) {
         case 'is_operator':
-            operation = contract.call('is_operator', address.toScVal());
+            operation = contract.call('is_operator', operator);
             break;
         case 'add_operator':
-            operation = contract.call('add_operator', address.toScVal());
+            operation = contract.call('add_operator', operator);
             break;
         case 'remove_operator':
-            operation = contract.call('remove_operator', address.toScVal());
+            operation = contract.call('remove_operator', operator);
+            break;
+
+        case 'refund': {
+            // eslint-disable-next-line no-case-declarations
+            const gasService = options.target || chain.contracts?.GasService?.address;
```

Comment:
```suggestion
            const gasService = options.target || chain.contracts?.axelar_gas_service?.address;
```
based on the contract name on stellar